neo/SmartContract/NefFile.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Neo.IO;
+using System;
+using System.IO;
+
+namespace Neo.SmartContract
+{
+    public class NefFile : ISerializable
+    {
+        public enum NefMagic : int
+        {
+            /// <summary>
+            /// NEO Executable Format 3
+            /// </summary>
+            NEF3 = 0x4E454633
+        }
+
+        /// <summary>
+        /// Magic
+        /// </summary>
+        public NefMagic Magic { get; set; }
+
+        /// <summary>
+        /// Compiler
+        /// </summary>
+        public string Compiler { get; set; }
+
+        /// <summary>
+        /// Version
+        /// </summary>
+        public Version Version { get; set; }
```

Comment:
Which version is this? Neo (core) version? NeoVM version? Perhaps both?
But since every Neo (core) release carries a specific NeoVM version, it's fine to just use Neo (core) version.

Reply:
(Mayor, Minor, Build, Version)